cartography/data/jobs/cleanup/azure_database_account_cleanup.json

```diff
@@ -1,82 +1,82 @@
 {
   "statements": [
     {
-      "query": "MATCH (n:AzureCosmosDBLocation)<-[:WRITE_PERMISSIONS_FROM]-(:AzureDatabaseAccount)<-[:RESOURCE]-(:AzureSubscription{id: $AZURE_SUBSCRIPTION_ID})<-[:RESOURCE]-(:AzureTenant{id: $AZURE_TENANT_ID})<-[:OWNER]-(:CloudanixWorkspace{id: $WORKSPACE_ID}) WHERE n.lastupdated <> $UPDATE_TAG WITH n LIMIT $LIMIT_SIZE DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
just to confirm, there's no relationship between database account and cosmosdb location?

Reply:
yes they have a relationship added in below line